packages/web-app-admin-settings/src/components/Users/SideBar/DetailsPanel.vue

```diff
@@ -81,6 +94,21 @@ export default defineComponent({
       return this.$gettext(
         this.roles.find((role) => role.id === assignedRole.appRoleId)?.displayName || ''
       )
+    },
+    groupsDisplayValue() {
+      return this.user.memberOf
+        .map((group) => group.displayName)
+        .sort()
+        .join(', ')
+    },
+    quotaDisplayValue() {
+      if (!('total' in (this.user.drive?.quota || {}))) {
+        return this.$gettext('Not set')
+      }
+
+      return this.user.drive.quota.total === 0
+        ? this.$gettext('No restriction')
+        : filesize(this.user.drive.quota.total)
```

Comment:
We have a nice helper for filesize: `formatFileSize`. It also respects the current language as the result may differ depending on it.